docs/source/starter/introduction_guide.rst

```diff
@@ -170,6 +207,9 @@ Pass in the dataloaders to the `.fit()` function.
 
 .. code-block:: python
 
+    from pytorch_lightning import Trainer
+
+
```

Comment:
Reviewers: Instead of having import statements scattered in the page, should we have a single place to import all functions/classes at the very top of this page maybe? Similarly to https://pytorch-lightning.readthedocs.io/en/latest/starter/new-project.html#step-0-install-pytorch-lightning

Reply:
Agree. The imports should be done once in the first example where they fit in and then not be repeated. If a new section appears that needs new imports, we add them.

The individual code snippets are anyway not runnable standalone and have to be taken with the context of the whole page.

Reply:
Thanks for this comment too!
I will move the import to the other imports location at the top of the section.